### ISSUE TITLE
Compilation fails: no type named 'GCEpilogueCallback' 

### ISSUE THREAD

**Issue by yurivict** [node-inspector/v8-profiler#92]
```
In file included from ../src/profiler.cc:2:
../node_modules/nan/nan.h:590:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:596:20: error: no type named 'GCEpilogueCallback' in 'v8::Isolate'
      v8::Isolate::GCEpilogueCallback callback) {
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:601:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback
      ~~~~~~~~~~~~~^
../node_modules/nan/nan.h:607:20: error: no type named 'GCPrologueCallback' in 'v8::Isolate'
      v8::Isolate::GCPrologueCallback callback) {
```

version-5.6.0
node-6.1.0
npm-3.8.8

**Comment by 3y3** [comment]
Should be fixed in latest version

**Comment by yurivict** [comment]
Sorry, but this isn't fixed in 5.6.0. Please re-open this bug.
